recipes/calour/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "calour" %}
+{% set version = "2018.05" %}
+{% set sha256 = "09b62a876e9bf32db996589362219b622feb0ca7b3a0739531e80fb2ae2b9246" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - matplotlib >=2.0.0
+    - scikit-learn >=0.19.1
+    - scikit-bio >=0.5.2
+    - biom-format
+    - statsmodels
+    - ipywidgets
+    - docrep
+test:
+  imports:
+    - calour
+    - calour.Experiment
+
+about:
+  home: https://biocore.github.io/calour/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: COPYING.txt
+  summary: 'exploratory and interactive microbiome analyses based on heatmaps'
+  description: |
+    Calour is a python package for processing, analysis and interactive
+    exploration of microbiome (and other matrix form data),
+    incorporating external databases.
+
+  doc_url: https://biocore.github.io/calour/
+  dev_url: https://github.com/biocore/calour
+
+extra:
+  recipe-maintainers:
+    - RNAer
+    - amnona
```

Comment:
Can you provide confirmation that this co-maintainer is willing to take on that role?